stdlib/public/core/StringBridge.swift

```diff
@@ -47,9 +47,8 @@ func _stdlib_binary_CFStringGetCharactersPtr(
 
 /// Bridges `source` to `Swift.String`, assuming that `source` has non-ASCII
 /// characters (does not apply ASCII optimizations).
-@_inlineable // FIXME(sil-serialize-all)
 @_versioned // FIXME(sil-serialize-all)
-@inline(never) @_semantics("stdlib_binary_only") // Hide the CF dependency
+@inline(never) // Hide the CF dependency
```

Comment:
IIRC The comment is solely from the lack of `@_inlineable`. I don't think it goes on `@inline(never)`, but I don't know where to put it. Perhaps under the new paradigm it's just not an interesting distinction. Thoughts?

Reply:
I think the comment is still useful for future maintainers in case they decide to put @_inlineable back in?

Reply:
In theory we shouldn't need the inline-never anymore either, right?

Reply:
Without inline-never it can be inlined inside the stdlib itself.

Reply:
Right, but that should be fine now.

Reply:
Outline-and-inline-never is still a useful, albeit niche, tool inside the stdlib when Builtin.expect is insufficient.